lib/datadog/core/configuration/settings.rb

```diff
@@ -654,6 +654,33 @@ def initialize(*_)
           end
         end
 
+        # The monotonic clock time provider used by Datadog.
```

Comment:
I think it is worth adding a comment to mention that `datadog-ci` is using this option to generate traces, not skewed by `Timecop` and we consider this internal (not public).